content/common/language-guide-overview.md

```diff
@@ -0,0 +1,448 @@
+title = "Language Support Overview"
+template = "common_main"
+date = "2022-01-01T00:00:01Z"
+enable_shortcodes = true
+[extra]
+
+---
+
+This page contains information about language support for Spin features:
+
+{{ tabs "sdk-type" }}
+
+{{ startTab "Rust"}}
+
+| Feature | SDK Supported? | Learn More |
+|-----|-----|-------|
+| **Triggers** |
+| HTTP | Supported | [Documentation](https://developer.fermyon.com/spin/http-trigger)<br/> [Code example](https://github.com/fermyon/spin/tree/main/examples/http-rust-outbound-http)<br/>Blog|
+| Redis | Supported | [Documentation](https://developer.fermyon.com/spin/redis-trigger)<br/> [Code example](https://github.com/fermyon/spin/tree/main/examples/redis-rust)<br/>Blog|
+| Custom | Supported | [Documentation](https://developer.fermyon.com/spin/extending-and-embedding)<br/>Code example<br/>Blog|
```

Comment:
Again I know this is not a review for content but custom triggers are not "supported" in Rust or anywhere else - almost by definition because once we support them they are no longer custom!  _grin_

(Or, per the link, is this more about _authoring_ custom triggers?  If so, maybe we should move the line to an "extensibility" or "other features" section - keep this section for using triggers _in applications._)

Reply:
Ah, I see, I will add another section to separate the different feature type.